src/appdistribution/client.ts

```diff
@@ -154,6 +196,30 @@ export class AppDistributionClient {
     return apiResponse.body;
   }
 
+  async listGroups(projectName: string): Promise<ListGroupsResponse> {
+    const groups: ListGroupsResponse = {
```

Comment:
Nit: Prefer calling this groupsResponse or something similar to avoid the awkward `groups.groups.blah` repetition

Reply:
tidied